src/coreclr/gc/handletablescan.cpp

```diff
@@ -992,6 +1001,23 @@ void BlockVerifyAgeMapForBlocksWorker(uint32_t *pdwGen, uint32_t dwClumpMask, Sc
                             }
                         }
                     }
+                    if (uType == HNDTYPE_WEAK_INTERIOR_POINTER)
+                    {
+                        PTR_uintptr_t pUserData = HandleQuickFetchUserDataPointer((OBJECTHANDLE)pValue);
+
+                        // if we did then copy the value
+                        if (pUserData)
+                        {
+                            TADDR pObjectInteriorPointer = **dac_cast<DPTR(DPTR(TADDR))>(pUserData);
```

Comment:
Nit: None of the surrounding code is DACized so dac_cast is unnecessary complication.

Reply:
Yeah, I thought that VerifyHeap worked in the DAC, but it turns out that it is an entirely parallel managed implementation these days. I'll pull that out if you'd like.

Reply:
There is a build break introduced by the `dac_cast` macro. My guess is that the easiest way to fix it is to delete the use here.

Reply:
Done